packages/docs/pages/Table/TablePage.tsx

```diff
@@ -315,6 +317,60 @@ const TablePage = () => {
         }}
         {/* jsx-to-string:end */}
       </CodePreview>
+
+      <H1>Usage with search</H1>
```

Comment:
Seems like the headers are slightly messed up on this page. This should probably be an `H2`.

Reply:
I changed only header with search. Maybe change others headers too (like in the `StatefulTablePage`)?

Reply:
That'd be great!